content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -124,10 +124,79 @@ You can also create a Markdown hyperlink by highlighting the text and using the
 
 {% data reusables.repositories.section-links %}
 
+If you need to determine the anchor for a heading in a file you are editing, you can use the following basic rules:
+
+* Letters are converted to lower-case.
+* Spaces are replaced by hyphens (`-`). Any other whitespace characters are removed.
+* Leading and trailing whitespace are removed.
+* Paired formatting markup characters are removed, leaving only their contents (for example, `_italics_` becomes `italics`).
```

Comment:
```suggestion
* Markup formatting is removed, leaving only the contents (for example, `_italics_` becomes `italics`).
```
I'm thinking about simplifying this in a way that not only Markdown formatting is removed, but basically any HTML formatting is removed … e.g. if the heading has a `<mark> foo </mark>` these would also not be part of the fragment I suppose…